rust/src/cxxrsutil.rs

```diff
@@ -55,7 +55,7 @@ macro_rules! impl_wrap {
             fn gobj_rewrap(&'a self) -> Pin<&'a mut Self::ReWrapped> {
                 // Access the underlying raw pointer behind the glib-rs
                 // newtype wrapper, e.g. `ostree_sys::OstreeRepo`.
-                let p: *mut $sys = self.to_glib_none().0;
+                let p: *const $sys = self.to_glib_none().0;
```

Comment:
This looks like it could be split into a prep commit.

Reply:
Ack, split out. Also submitted as its own PR at https://github.com/coreos/rpm-ostree/pull/2929 in order to get a sanity check from CI.